src/particles/elements/Elemlist.H

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Axel Huebl, Ji Qiang
```

Comment:
We can remove this with `git rm src/particles/elements/Elemlist.H` and then `git commit` it.